test/e2e/common/downward_api.go

```diff
@@ -217,6 +293,36 @@ func testDownwardAPI(f *framework.Framework, podName string, env []v1.EnvVar, ex
 	testDownwardAPIUsingPod(f, pod, env, expectations)
 }
 
+func testDownwardAPIForEphemeralStorage(f *framework.Framework, podName string, env []v1.EnvVar, expectations []string) {
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   podName,
+			Labels: map[string]string{"name": podName},
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:    "dapi-container",
+					Image:   busyboxImage,
+					Command: []string{"sh", "-c", "env"},
+					Resources: v1.ResourceRequirements{
+						Requests: v1.ResourceList{
+							v1.ResourceEphemeralStorage: resource.MustParse("32Mi"),
+						},
+						Limits: v1.ResourceList{
+							v1.ResourceEphemeralStorage: resource.MustParse("64Mi"),
+						},
+					},
+					Env: env,
+				},
+			},
+			RestartPolicy: v1.RestartPolicyNever,
+		},
+	}
+
+	testDownwardAPIUsingPod(f, pod, env, expectations)
+}
+
 func testDownwardAPIUsingPod(f *framework.Framework, pod *v1.Pod, env []v1.EnvVar, expectations []string) {
```

Comment:
where is `env` used here?

Reply:
yeah, seems not used, it is not added by me, i will delete it

Reply:
I still see `env`...

Reply:
This comment appears to not have been addressed.

Reply:
Yeah, this is not introduced by me. I will look into that and send a separate pr to solve it